pettingzoo/classic/tictactoe/tictactoe.py

```diff
@@ -221,34 +247,74 @@ def reset(self, seed=None, options=None):
         self._agent_selector.reset()
         self.agent_selection = self._agent_selector.reset()
 
+    def close(self):
+        pass
+
     def render(self):
         if self.render_mode is None:
             gymnasium.logger.warn(
                 "You are calling render method without specifying any render mode."
             )
             return
 
+        screen_height = self.screen_height
+        screen_width = self.screen_height
+
+        # TODO: refactor this and check if pygame.font init needs to be done
```

Comment:
Minor but if you could look into this TODO and then ideally remove it from all the different files (I see it's in all the new ones you added, my assumption is it's either your own TODO or from texas holdem or something else, either way good to resolve)

Reply:
Removed the ToDo for now. I see that the `pygame.font.init()` does not do anything, since rgb_array renders are not supported by these environments.